backend/src/services/utils/authenticated_user.py

```diff
@@ -1,24 +1,30 @@
 # pylint: disable=bare-except
 
-from typing import Any, Optional
+from typing import Any, Optional, TypedDict
+
+
+class AccessTokens(TypedDict):
+    graph_access_token: Optional[str]
+    sumo_access_token: Optional[str]
+    smda_access_token: Optional[str]
+    pdm_access_token: Optional[str]
+    ssdl_access_token: Optional[str]
 
 
 class AuthenticatedUser:
     def __init__(
         self,
         user_id: str,
         username: str,
-        sumo_access_token: Optional[str],
-        smda_access_token: Optional[str],
-        pdm_access_token: Optional[str],
-        ssdl_access_token: Optional[str],
+        access_tokens: AccessTokens,
     ) -> None:
         self._user_id = user_id
         self._username = username
-        self._sumo_access_token = sumo_access_token
-        self._smda_access_token = smda_access_token
-        self._pdm_access_token = pdm_access_token
-        self._ssdl_access_token = ssdl_access_token
+        self._graph_access_token = access_tokens.get("graph_access_token", None)
+        self._sumo_access_token = access_tokens.get("sumo_access_token", None)
+        self._smda_access_token = access_tokens.get("smda_access_token", None)
+        self._pdm_access_token = access_tokens.get("pdm_access_token", None)
+        self._ssdl_access_token = access_tokens.get("ssdl_access_token", None)
```

Comment:
```suggestion
        self._graph_access_token = access_tokens.get("graph_access_token")
        self._sumo_access_token = access_tokens.get("sumo_access_token")
        self._smda_access_token = access_tokens.get("smda_access_token")
        self._pdm_access_token = access_tokens.get("pdm_access_token")
        self._ssdl_access_token = access_tokens.get("ssdl_access_token")
```
I think `None` is default value if key does not exist?